### PR TITLE
Framework for Bat Wings

### PR DESCRIPTION
**Pull request by Ignose**
Requires:
1. Better understanding of free fight rate
2. Mafia tracking of free fights (limit 5)
3. Mafia tracking of bat skills
4. lint update

### CODE REVIEW

File: packages/garbo/src/fights.ts [hunk 1]
```diff
@@ -1297,6 +1299,8 @@ const freeRunFightSources = [
           .trySkillRepeat(
             $skill`Fire Extinguisher: Polar Vortex`,
             $skill`Perpetrate Mild Evil`,
+            // eslint-disable-next-line libram/verify-constants
+            $skill`Swoop like a Bat`,
```
Comment (Rinn) [inline packages/garbo/src/fights.ts:1303]: are we nearing the round limit with this?

Comment (Ignose) [reply packages/garbo/src/fights.ts:1303]: I don't know. We could be. The plan is to test it tomorrow and find out.

Comment (Ignose) [reply packages/garbo/src/fights.ts:1303]: To answer your question, it didn't swoop! So I still don't know. I think Libram/eslint need to be updated first to test that.

Comment (horrible-little-slime) [reply packages/garbo/src/fights.ts]: No, you just never wrote code to equip the wings.

File: packages/garbo/src/tasks/barfTurn.ts [hunk 2]
```diff
@@ -95,6 +97,9 @@ import { lavaDogsAccessible, lavaDogsComplete } from "../resources/doghouse";
 import { hotTubAvailable } from "../resources/clanVIP";
 import { meatMood } from "../mood";
 
+let batWingsTurns = 0;
+let batWingTriggers = 0;
```
Comment (horrible-little-slime) [inline packages/garbo/src/tasks/barfTurn.ts:101]: This doesn't belong here, it belongs in excavator

**Pull request merged**